aries-site/src/examples/templates/table-customization/components/TableCustomizationExample.js

```diff
@@ -92,87 +89,58 @@ const filtersConfig = [
-                  <FilterControls
-                    // Table column configuration should be grouped on right
-                    // side with other actions to separate it from filtering.
-                    // If grouped to the left with the Filter control, it
-                    // becomes confusing what the "Clear filters" button will
-                    // do with regards to any column configurations that have
-                    // been applied.
-                    configure={
-                      <DropButton
-                        a11yTitle="Configure columns button"
-                        icon={<Splits />}
-                        kind="toolbar"
-                        dropAlign={{ top: 'bottom', right: 'right' }}
-                        onClose={() => setOpen(false)}
-                        onOpen={() => setOpen(true)}
-                        dropContent={
-                          <ColumnSettings
-                            columns={COLUMNS}
-                            visibleColumns={visibleColumns}
-                            setVisibleColumns={setVisibleColumns}
-                            open={open}
-                          />
-                        }
-                        tip="Configure columns"
-                      />
-                    }
-                    actions={<Menu kind="toolbar" label="Actions" items={[]} />}
-                    data={allData}
-                    filters={filtersConfig}
-                    searchFilter={{ placeholder: 'Search' }}
-                  />
-                </Box>
-              </Box>
-            </Header>
-            <Results columns={visibleColumns} />
+export const TableCustomizationExample = () => (
+  <Page background="background" fill>
+    <PageContent>
+      <Box gap="medium">
+        <Header pad={{ top: 'medium' }}>
+          <Box gap="xsmall" fill="horizontal">
+            <Heading id="users-heading" level={2} margin="none">
+              Users
+            </Heading>
           </Box>
-        </FiltersProvider>
-      </PageContent>
-    </Page>
-  );
-};
+        </Header>
+        <Results />
+      </Box>
+    </PageContent>
+  </Page>
+);
 
-const Results = ({ columns }) => {
+const Results = () => {
   const [select, setSelect] = useState([]);
-  const { filteredResults } = useFilters();
 
   return (
-    <Box fill overflow="auto">
-      <DataTable
-        aria-describedby="users-heading"
-        data={filteredResults}
-        background="background"
-        columns={columns}
-        select={select}
-        onSelect={setSelect}
-        pin
-      />
+    <Box fill>
+      <Data data={allData} updateOn="change" fill>
+        <Toolbar>
+          <Box direction="column" fill>
+            <Box direction="row" gap="small">
+              <DataSearch />
+              <DataTableColumns drop options={filtersConfig} />
+              <DataFilters drop />
+              <Box fill />
```

Comment:
```suggestion
              <Box flex />
```
I think this better mirrors patterns I've seen